python/ray/data/_internal/util.py

```diff
@@ -24,6 +24,7 @@
 MAX_PYARROW_VERSION = "7.0.0"
 RAY_DISABLE_PYARROW_VERSION_CHECK = "RAY_DISABLE_PYARROW_VERSION_CHECK"
 _VERSION_VALIDATED = False
+_LOCAL_SCHEME = "local://"
```

Comment:
What happens if `local:/file/path` is specified? (one slash)?

Reply:
It was not regarded as local scheme before; with url parser, it's recognized as `/file/path`.